src/pages/partners/_partnership-opportunities.js

```diff
@@ -68,62 +69,123 @@ const ClientCard = styled(LocalizedLink)`
-            </ClientCard>
-        </Flex>
-    </SectionContainer>
-)
+const PartnershipOpportunities = () => {
+    const { is_row } = useCountryRule()
+    return (
+        <SectionContainer>
+            <Container direction="column">
+                <Header align="center" size="var(--text-size-header-1)" as="h2" mb="4rem">
+                    {localize('Partnership opportunities')}
+                </Header>
+            </Container>
+            <Flex wrap="wrap">
+                <ClientCard to="/partners/affiliate-ib/" first>
+                    <Flex ai="center" height="auto" mb="0.8rem">
+                        <Header as="h4" size="var(--text-size-sm)">
+                            {localize('Affiliate and IB programmes')}
+                        </Header>
+                        <PartnerAffiliate />
+                    </Flex>
+                    <Text>
+                        {localize(
+                            'Enjoy the flexibility to choose from a variety of affiliate and introducing broker programmes. Earn 45% commission from promoting our trading platforms or enjoy turnover-based commissions.',
+                        )}
+                    </Text>
+                </ClientCard>
+                {is_row && (
+                    <ClientCard to="/partners/payment-agent/" second>
+                        <Flex ai="center" height="auto" mb="0.8rem">
+                            <Header as="h4" size="var(--text-size-sm)">
+                                {localize('Payment agent programme')}
+                            </Header>
+                            <PartnerPaymentAgent />
+                        </Flex>
+                        <Text>
+                            {localize(
+                                'Expand your client base by helping the traders who are looking for ways to fund their accounts through local bank wires and e-payment methods.',
+                            )}
+                        </Text>
+                    </ClientCard>
+                )}
+
+                <ClientCard
+                    to=""
+                    type="api"
+                    target="_blank"
+                    external="true"
+                    rel="noopener noreferrer"
+                    third
+                >
+                    <Flex ai="center" height="auto" mb="0.8rem">
+                        <Header as="h4" size="var(--text-size-sm)">
+                            {localize('API')}
+                        </Header>
+                        <DeveloperProgramme />
+                    </Flex>
+                    <Text>
+                        {localize(
+                            'Launch your trading app powered by our APIs. Get up to 5% in markup earnings from every trade your clients make on your app.',
+                        )}
+                    </Text>
+                </ClientCard>
+            </Flex>
+        </SectionContainer>
+    )
+}
+;<SectionContainer>
```

Comment:
this should not be here I guess, this is causing the checks issue. @meenakshi-deriv